src/main/java/svc/security/HashUtil.java

```diff
@@ -0,0 +1,59 @@
+package svc.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HashUtil{
+	@Value("${stlcourts.security.municipalitySalt}")
+	String municipalitySalt;
+	@Value("${stlcourts.security.judgeSalt}")
+	String judgeSalt;
+	@Value("${stlcourts.security.courtSalt}")
+	String courtSalt;
+	
+	public Hashids municipalityHashids;
+	public Hashids judgeHashids;
+	public Hashids courtHashids;
+	
+	private void initializeSalts(){
```

Comment:
So I found that @Value is only read at runtime off of an @Inject.  If I try to use the salt values outside of a method, they return null.  Same goes if I create a constructor they are still null there.  So only during runtime when a HashUtil.encode is called are the @Value salts valid.